name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dashboard/SharpeRatioWidget.java

```diff
@@ -0,0 +1,37 @@
+package name.abuchen.portfolio.ui.views.dashboard;
+
+import name.abuchen.portfolio.model.Dashboard.Widget;
+import name.abuchen.portfolio.money.Values;
+import name.abuchen.portfolio.snapshot.PerformanceIndex;
+import name.abuchen.portfolio.ui.Messages;
+
+/**
+ * Display the current Sharpe Ratio of the Portfolio
+ */
+public class SharpeRatioWidget extends IndicatorWidget<Double>
+{
+    public SharpeRatioWidget(Widget widget, DashboardData dashboardData)
+    {
+        super(widget, dashboardData, true, null);
+        SharpeRatioConfig configuration = new SharpeRatioConfig(this);
+        this.addConfig(configuration);
+
+        this.setFormatter(Values.PercentPlain);
+        this.setValueColored(false);
+        this.setTooltip((ds, period) -> {
+            return Messages.TooltipSharpeRatio;
+        });
+        this.setProvider((ds, period) -> {
+            PerformanceIndex index = dashboardData.calculate(ds, period);
+            double r = index.getPerformanceIRR();
+            double rf = ((double)configuration.getRiskFreeIRR()) / 10000;
+            double volatility = index.getVolatility().getStandardDeviation();
+
+            if (Double.isNaN(rf))
+                return Double.NaN; // Handle invalid rf value
+
+            double excessReturn = r - rf;
+            return excessReturn / volatility;
```

Comment:
In the SharpeRatioWiget file they call the variable e.g. “riskFreeIRR” and here only "r" or “rf”... 
Let's name it immediately 👍🏻 

```suggestion
            double performanceIRR = index.getPerformanceIRR();
            double riskFreeIRR = ((double)configuration.getRiskFreeIRR()) / 10000;
            double volatility = index.getVolatility().getStandardDeviation();

            if (Double.isNaN(riskFreeIRR) || volatility == 0)
                return Double.NaN; // Handle invalid risk-free IRR or zero volatility

            double excessReturn = performanceIRR - riskFreeIRR;
            return excessReturn / volatility;
```